### ISSUE TITLE
possible to use own code to attach process?

### ISSUE THREAD

**Issue by aDioS786** [cheat-engine/cheat-engine#1651]
Hi
Is it possible to use DMA hardware to attach / update memory in CE? If yes can you give guidance on how it might be possible?
Currently it directly attached to process, whereas it be great to change it slightly so rather attaching directly to process, it attached/read process via DMA device. Thx

**Comment by cheat-engine** [comment]
Yes.  Open the [Physical Memory] process.  It shows when kernelmode read/write processmemory is checked in settings, or when dbvm is loaded

**Comment by aDioS786** [comment]
Thanks a lot for prompt response can you explain a bit more please..

**Comment by l3arnmore** [comment]
> Thanks a lot for prompt response can you explain a bit more please..

literally. if you have loaded DBVM succesfully process list would display [Physical Memory]. after opening it you can DMA as you want.
